app/feature/feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/step/choose/ChooseInsuranceToTerminateDestination.kt

```diff
@@ -140,43 +137,18 @@ private fun ChooseInsuranceToTerminateScreen(
             Spacer(Modifier.height(16.dp))
           }
         }
-        for (insurance in uiState.insuranceList) {
-          HedvigCard(
-            onClick = { selectInsurance(insurance) },
-            colors = CardDefaults.outlinedCardColors(
-              containerColor = MaterialTheme.colorScheme.surface,
-            ),
+        val radioOptionData = uiState.insuranceList.toRadioOptionDataList(uiState.selectedInsurance?.id)
+        com.hedvig.android.design.system.hedvig.HedvigTheme {
+          // todo: where do we apply the theme now that we're still on both old and new theme?
```

Comment:
So at the top-level where we provide it for the entire app, we need to provide both themes.
Take a look at this https://github.com/HedvigInsurance/android/blob/5001fea1697470018ffd9a7014c3a8a24acc14b7/app/app/src/main/kotlin/com/hedvig/android/app/ui/HedvigApp.kt#L131-L139 I seem to already have done that

Reply:
Ah, alright! Maybe it was because I've seen it in HedvigPreview, which is the old theme, and it was not behaving well there